tests/localnet/conftest.py

```diff
@@ -13,3 +18,57 @@ def algod_health_fast_timings(mocker: MockerFixture) -> None:  # noqa: ignore[PT
 @pytest.fixture()
 def health_success(httpx_mock: HTTPXMock) -> None:  # noqa: ignore[PT004]
     httpx_mock.add_response(url=ALGOD_HEALTH_URL)
+
+
+@pytest.fixture()
+def localnet_up_to_date(proc_mock: ProcMock, httpx_mock: HTTPXMock) -> None:
+    proc_mock.set_output(
+        ["docker", "image", "inspect", "algorand/algod:latest", "--format", "{{.RepoDigests}}"],
```

Comment:
If we make the image tags constants as mentioned earlier, then they can also be referenced here in the tests instead of defining them in multiple places